inputs/wysiwyg.vue

```diff
@@ -184,7 +199,15 @@
         </div>
       </div>
     </div>
-    <div v-else class="wysiwyg-content"></div>
+    <div v-else></div>
+      <div class="wysiwyg-content"></div>
+      <div v-if="isTooLong" :class="[
+        'ui-textbox__counter', 
+        'ui-textbox__counter--wysiwyg',
+        {'ui-textbox__counter--error': isTooLong} ]">
+        {{ valueLength + '/' + maxLength }}
```

Comment:
@nelsonpecora `textbox-counter` could potentially be broken out as a micro-component. Thoughts?

Reply:
Hmm, not sure we need to for now, but good to keep in mind if we start using this pattern in other places